modules/nf-core/delly/call/main.nf

```diff
@@ -1,35 +1,37 @@
 process DELLY_CALL {
     tag "$meta.id"
-    label 'process_medium'
+    label 'process_single'
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/delly:1.1.6--ha41ced6_0' :
-        'biocontainers/delly:1.1.6--ha41ced6_0' }"
+        'https://depot.galaxyproject.org/singularity/delly:1.2.6--hb7e2ac5_0':
+        'biocontainers/delly:1.2.6--hb7e2ac5_0' }"
 
     input:
-    tuple val(meta), path(input), path(input_index), path(vcf), path(vcf_index), path(exclude_bed)
-    path fasta
-    path fai
+    tuple val(meta), path(input_1), path(input_1_idx), path(input_ctrl), path(input_ctrl_idx), path(exclude_bed)
+    tuple val(ref_meta), path(fasta), path(fai)
+    tuple val(vcf_meta), path(vcf), path(vcf_index)
```

Comment:
```suggestion
    tuple val(meta2), path(fasta)
    tuple val(meta3), path(fai)
    tuple val(meta4), path(vcf), path(vcf_index)
```
To stay consistent with all other modules. Also keep in mind that the first channel always needs to have a valid meta map since this is used for a lot of defaults. Maybe it would be better to put the VCFs back in there?

Reply:
The vcf file contains a common call set that is usually shared across many samples, so I thought it would be more convenient to be able to provide that as a separate value channel. I can put it back into the first input channel tuple to keep changes as small as possible.

Reply:
Ah I get it, I thought these VCFs would be used for the genotype functionality of `delly call`

Reply:
I changed the naming of the meta maps, separated `fasta` and `fai` into two channels and also added `vcf `and `vcf_index` to the first channel to keep changes as small as possible.